src/pages/tutorials/basic-modifications/index.md

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
When you build, are you able to get to this page through the side navigation?

Reply:
Ah...Needed links to the new overview pages. Thanks.